src/kde/kde_pt.cc

```diff
@@ -0,0 +1,24 @@
+//--------------------------------------------*-C++-*---------------------------------------------//
+/*!
+ * \file   kde/kde_pt.cc
+ * \author Mathew Cleveland
+ * \date   Nov. 10th 2020
+ * \brief  Explicit template instatiations for class kde.
+ * \note   Copyright (C) 2018-2020 Triad National Security, LLC.
+ *         All rights reserved.
+ *
+ */
+//------------------------------------------------------------------------------------------------//
+
+#include "kde.t.hh"
+
+namespace kde {
+
+// Explicit template instantiations go here.
+template class kde<kde_coordinates::CART>;
```

Comment:
This only makes sense if the definition in the `.t.hh` file is for `kde<coord>`.  As it is, the `.t.hh` has no unassigned templates and so is no different that a regular function in a `.cc` file.  This is fine if the development is taking you in the right direction, but if the kde object and members functions will always be fully defined, then we need to change the file layout.

Reply:
I see. I think this would imply that I do not need the _pt file and the .t.hh should be a .cc. Correct?